src/pages/RunEncounter/DisplayInitiative.tsx

```diff
@@ -58,8 +59,17 @@ const DisplayInitiative: React.FunctionComponent = () => {
 
   return (
     <div>
-      <Button onClick={() => dispatch(open())}>Quick Add Character</Button>
-      <QuickAddModal />
+      <Row>
+        <Col md="auto">
+          <Button onClick={() => dispatch(open())}>Quick Add Character</Button>
+        </Col>
+        <Col md="auto">
+          <Button onClick={() => dispatch(rollAllInitiative())}>Roll Initiative</Button>
+        </Col>
+      </Row>
+      <Row>
+        <QuickAddModal />
+      </Row>
```

Comment:
Since there's only one row (modal doesn't need a row), the full bootstrap container functionality isn't needed here and can be simplified as:

```tsx
<Stack direction="horizontal" gap={2}>
  <Button onClick={() => dispatch(open())}>Quick Add Character</Button>
  <Button onClick={() => dispatch(rollAllInitiative())}>Roll Initiative</Button>
</Stack>
```